app/src/docs/_examples/rangeDatePicker/PresetsAndFooter.example.tsx

```diff
@@ -28,13 +28,13 @@ export default function DatePickerBaseExample() {
                         },
                     },
                 } }
-                renderFooter={ (value: RangeDatePickerValue) => (
+                renderFooter={ (footerValue: RangeDatePickerValue) => (
```

Comment:
Footer doesn't have any value, it this case this is the current state of DatePicker, it's will confuse, let's change the name of param. IMHO the previous one was ok.